server/src/main/java/org/apache/iotdb/db/qp/physical/sys/MergePlan.java

```diff
@@ -38,4 +38,9 @@ public MergePlan() {
   public List<Path> getPaths() {
     return new ArrayList<>();
   }
+
+  @Override
+  public List<String> getPathsStrings() {
+    return new ArrayList<>();
```

Comment:
And here.

Reply:
I'll change it. Thanks for your suggestion.